### PR TITLE
FRS documentation with TokenProvider and Azure function

### PR DESCRIPTION
**Pull request by sonalivdeshpande**
This is draft PR for FRS `FrsAzFunctionTokenProvider` documentation. It includes setting up the function to fetching the FRS token on the client-side.

### CODE REVIEW

File: docs/content/docs/deployment/azure-frs.md [hunk 1]
```diff
@@ -33,7 +33,36 @@ Now that you have an instance of `FrsClient`, you can start using it to create o
 
 ### Token providers
 
-{{< placeholder >}}
+The [FrsAzFunctionTokenProvider]({{< relref "https://github.com/microsoft/FluidFramework/blob/main/experimental/framework/frs-client/src/FrsAzFunctionTokenProvider.ts" >}}) is an implementation of `ITokenProvider` which ensures your tenant key secret is not exposed in your client-side bundle code. The `FrsAzFunctionTokenProvider` takes in your Azure Function URL appended by `/api/GetFrsToken` along with the current user object. Later on, it makes an axios `GET` request call to your Azure function by passing in the tenantID, documentId and userID/userName as optional parameters.
+
+```javascript
+const config = {
+    tenantId: "myTenantId",
+    tokenProvider: new FrsAzFunctionTokenProvider("AZURE-FUNCTION-URL"+"/api/GetFrsToken", { userId:
```
Comment (sumedhb1995) [inline docs/content/docs/deployment/azure-frs.md:41]:
```suggestion
    tokenProvider: new FrsAzFunctionTokenProvider("myAzureFunctionUrl"+"/api/GetFrsToken", { userId:
```

Comment (sumedhb1995) [reply docs/content/docs/deployment/azure-frs.md:41]: just to be consistent with the other strings here 😄

File: docs/content/docs/deployment/azure-frs.md [hunk 2]
```diff
@@ -33,7 +33,36 @@ Now that you have an instance of `FrsClient`, you can start using it to create o
...
+    "UserId", userName: "Test User"}),
+    orderer: "https://myOrdererUrl",
+    storage: "https://myStorageUrl",
+};
+
+const client = new FrsClient(config);
+```
+The user object can also hold optional additional user details such as the gender, address, email, etc.
```
Comment (sumedhb1995) [inline docs/content/docs/deployment/azure-frs.md:49]:
```suggestion
The user object can also hold optional additional user details such as the gender, address, email, etc. For example:
```

File: docs/content/docs/deployment/azure-frs.md [hunk 3]
```diff
@@ -33,7 +33,36 @@ Now that you have an instance of `FrsClient`, you can start using it to create o
...
+
+```javascript
+cont userDetails = {
+  email: "xyz@outlook.com",
+  address: "Redmond",
+};
+
```
Comment (sumedhb1995) [inline docs/content/docs/deployment/azure-frs.md:56]: Let's also specify that the developer can add in any other fields here for this object that they would like to see returned in the `FrsMember` values from the audience in container services. I think a code sample showing the same additionalDetails field in the example above being parsed from audience would be helpful to show the entire loop

File: docs/content/docs/deployment/azure-frs.md [hunk 4]
```diff
@@ -105,22 +105,30 @@ const { audience } = containerServices;
...
+            Current User: ${self?.userName} <br />
+            Other Users: ${memberStrings.join(", ")}
+        `;
+    };
+
+    onAudienceChanged();
+    audience.on("membersChanged", onAudienceChanged);
 ```
 
 `audience` provides two functions that will return `FrsMember` objects that have a user ID and user name:
```
Comment (sumedhb1995) [inline docs/content/docs/deployment/azure-frs.md:134]: I believe we also need to update the sample `FrsMember` value below to include the additionalDetails now

Comment (sumedhb1995) [reply docs/content/docs/deployment/azure-frs.md:134]: Also, we should specifically state that the values in those fields match up to the values provided in the token

Comment (sdeshpande3) [reply docs/content/docs/deployment/azure-frs.md:134]: Yep! Sorry missed that

File: docs/content/docs/deployment/azure-frs.md [hunk 5]
```diff
@@ -116,11 +153,15 @@ A sample `FrsMember` object looks like the following:
...
+Alongside the user ID, name and addiitonalDetails, `FrsMember` objects also hold an array of `connections`. If the user is logged into the session with only one client, `connections` will only have one value in it with the ID of the client and if is in read/write mode. However, if the same user is logged in from multiple clients (i.e. they are logged in from different devices or have multiple browser tabs open with the same container), `connections` here will hold multiple values for each client. In the example data above, we can see that a user with name "Test User" and ID "0e662aca-9d7d-4ff0-8faf-9f8672b70f15" currently has the container open from two different clients. The values in the `additionalDetails` field match up to the values provided in the `FrsAzFunctionTokenProvider` token generation.
```
Comment (sumedhb1995) [inline docs/content/docs/deployment/azure-frs.md:164]:
```suggestion
Alongside the user ID, name and additional details, `FrsMember` objects also hold an array of `connections`. If the user is logged into the session with only one client, `connections` will only have one value in it with the ID of the client and if is in read/write mode. However, if the same user is logged in from multiple clients (i.e. they are logged in from different devices or have multiple browser tabs open with the same container), `connections` here will hold multiple values for each client. In the example data above, we can see that a user with name "Test User" and ID "0e662aca-9d7d-4ff0-8faf-9f8672b70f15" currently has the container open from two different clients. The values in the `additionalDetails` field match up to the values provided in the `FrsAzFunctionTokenProvider` token generation.
```

**Review by sumedhb1995 — APPROVED** [review]
🚢

**Pull request merged**